e2e/testdata/simple-bundle.json.golden

```diff
@@ -0,0 +1,118 @@
+{
+	"name": "acmecorp/simple",
+	"version": "1.1.0-beta1",
+	"description": "new fancy webapp with microservices",
+	"maintainers": [
+		{
+			"name": "John Developer",
+			"email": "john.dev@acmecorp.cool",
+			"url": ""
+		},
+		{
+			"name": "Jane Developer",
+			"email": "jane.dev@acmecorp.cool",
+			"url": ""
+		}
+	],
+	"invocationImages": [
+		{
+			"imageType": "docker",
+			"image": "acmecorp/simple:1.1.0-beta1-invoc"
+		}
+	],
+	"images": {
+		"api": {
+			"imageType": "docker",
+			"image": "python:3.6",
+			"description": "python:3.6",
+			"refs": null
+		},
+		"db": {
+			"imageType": "docker",
+			"image": "postgres:9.3",
+			"description": "postgres:9.3",
+			"refs": null
+		},
+		"web": {
+			"imageType": "docker",
+			"image": "nginx:latest",
+			"description": "nginx:latest",
+			"refs": null
+		}
+	},
+	"actions": {
+		"inspect": {
+			"Modifies": false
```

Comment:
Shouldn't this be `"modifies"`? How come everything works, JSON keys should be case sensitive...

Reply:
I think that's the bundle struct in go which does not rename the field for serialization. So it works but it's not compliant with the spec. I'll open a PR against duffle.